app/lib/world/world.coffee

```diff
@@ -723,6 +723,9 @@ module.exports = class World
     return unless level.constrainHeroHealth
     hero = _.find @thangs, id: 'Hero Placeholder'
     if hero?
+      for prop, {min, max} of level.clampedProperties ? {}
+        hero[prop] = Math.min hero[prop], max if max?
```

Comment:
add parentheses or comments so that it won't confused saurabh or adam that it is `(Math.min hero[prop], max) if max?`